app/helpers/publishing_helper.rb

```diff
@@ -0,0 +1,11 @@
+module PublishingHelper
+  def remote_publish!(&block)
```

Comment:
This name feels overloaded - it's not just being used for publishing (in terms of talking to the publishing api) but also for talking to Rummager and email-alert-api.